src/design-system/tokens.scss

```diff
@@ -0,0 +1,16 @@
+:root {
+  --x-color-lead: #36515b;
```

Comment:
The default colors are:

```
  --x-color-lead: #36515b;
  --x-color-auxiliary: #667981;
  --x-color-neutral-10: #1a1a1a;
  --x-color-neutral-35: #595959;
  --x-color-neutral-70: #b3b3b3;
  --x-color-neutral-95: #f2f2f2;
  --x-color-neutral-100: #ffffff;
```

So I think you can delete 

- --x-color-lead
-  --x-color-neutral-10
-  --x-color-neutral-35
-  --x-color-neutral-100